docs/function/handlers.md

```diff
@@ -224,6 +224,21 @@ Note that path parameters (e.g. `{id}` in the example above) are available as re
 $id = $request->getAttribute('id');
 ```
 
+### Event and context
```

Comment:
Please add a empty line after the heading

Reply:
```suggestion
### Lambda event and context
```